SkillChat.Server/NETVersionComparison.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+
+namespace SkillChat.Server
+{
+    public class NETVersionComparison
+    {
+        /// <summary>
+        /// Получение наиболее подходящего Runtime .NET
+        /// </summary>
+        public static Runtime GetBestDotNETRuntime(string cfgVersion)
+        {
+            Runtime cfgRuntime = new Runtime() {Version = cfgVersion};
+            List<Runtime> _runtimesInstaled = new List<Runtime>();
+            List<Runtime> _NETCoreRuntimes = new List<Runtime>();
+
+            GetPackageList(_runtimesInstaled);
+
+            foreach (var item in _runtimesInstaled)
+            {
+                if (item.Name == "Microsoft.NETCore.App")
+                {
+                    _NETCoreRuntimes.Add(item);
+                }
+            }
+
+            if (_NETCoreRuntimes.Count > 0)
+            {
+                _NETCoreRuntimes.Sort((Runtime version1, Runtime version2) =>
+                {
+                    if (version1.Version == null && version2.Version == null) return 0;
+                    else if (version1.Version == null) return 1;
+                    else if (version2.Version == null) return -1;
+                    else return version1.Version.CompareTo(version2.Version);
+                });
+
+                foreach (var VARIABLE in _NETCoreRuntimes)
```

Comment:
Название VARIABLE неинформативное и нарушает конвенцию именования тем, что оно большими буквами.
Должно быть что-то типа runtime
```suggestion
                foreach (var runtime in _NETCoreRuntimes)
```